src/Controller/GameController.php

```diff
@@ -84,7 +96,7 @@ public function joinGame(Request $request, string $id, PublisherInterface $publi
     {
         $content = $request->toArray();
         if (!$content || !$content["playerId"]) {
-            return new JsonResponse("Id and playerId are required", 400);
+            return new Response("Id and playerId are required", 400);
```

Comment:
Same: Does this return json? If not, it can be tricky for api customers to have different return formats in case of success or failure